engine/docs/src/script_doc.py

```diff
@@ -192,25 +206,33 @@ def _parse_comment(str):
             document_comment = True
 
     if not name_found:
-        logging.warn('Missing tag @name in "%s"' % str)
+        logging.warn('Missing tag @name in "%s"' % text)
```

Comment:
Took _waaaayyy_ too long to realize why the code changes failed mysteriously. Just because we used an actual typename, so when I added "isinstance(text, str)" everything broke :(